From 8328236a7a1a0de6cb70129b1a31304bc7078751 Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Tue, 14 Aug 2018 19:33:50 -0700
Subject: [PATCH 11/13] [TEMP] LineageSettingsProvider: Do not access system
 settings during startup

*) This is not possible since:
https://android.googlesource.com/platform/frameworks/base/+/235272a2165b694ca1d67e901aa2ef2ad86c35bc

*) Placeholder commit to allow LineageSettingsProvider to start

Change-Id: I185648ca25995fae80362642d297573795d04bfa
---
 .../org/lineageos/lineagesettings/LineageDatabaseHelper.java    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
index f245dd9..c74ddfa 100644
--- a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
+++ b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
@@ -420,9 +420,11 @@ public class LineageDatabaseHelper extends SQLiteOpenHelper{
                     LineageSettings.Secure.PROTECTED_COMPONENT_MANAGERS,
                     R.string.def_protected_component_managers);
 
+            /*
             final String provisionedFlag = Settings.Global.getString(mContext.getContentResolver(),
                     Settings.Global.DEVICE_PROVISIONED);
             loadSetting(stmt, LineageSettings.Secure.LINEAGE_SETUP_WIZARD_COMPLETED, provisionedFlag);
+            */
         } finally {
             if (stmt != null) stmt.close();
         }
-- 
2.17.1

